### DISCUSSION TITLE
Create 3 Security Robot

### DISCUSSION THREAD

**Discussion by leord22** [iRobotEducation/create3_docs#365] [Show and tell]
Hello!

Thank you very much to the Create3 forum admins and community for providing valuable information on Create3 setup and configuration. I'd also like to share what I've built. I apologize for the delay in posting, since this was done a couple of months ago, but I was occupied with my current projects. Here is the link to my video. Feel free to leave comments or suggestions in the comments section.
https://www.youtube.com/watch?v=-vJ61rwWhts

Thank you!

**Comment by shamlian** [comment]
We love hearing about the cool stuff that you're building; this video I'm sure will be a great resource for others looking to build a similar platform and I appreciate you taking the time to put it together and post it!

**Reply by leord22** [reply]
Thank you very much for your feedback! Finally I've created the second part of the video, which has more details on how to build this security robot: https://youtu.be/bsgij1WIVX4 
Please let me know if you have any questions or comments; they are always welcome!
